packages/repack/README.md

```diff
@@ -27,14 +27,12 @@ Like the project? ⚛️ [Join the team](https://callstack.com/careers/?utm_camp
 
 <!-- badges -->
 
-[callstack-readme-with-love]: https://callstack.com/?utm_source=github.com&utm_medium=referral&utm_campaign=react-native-paper&utm_term=readme-with-love
-[build-badge]: https://img.shields.io/github/workflow/status/callstack/repack/CI/main?style=flat-square
-[build]: https://github.com/callstack/repack/actions/workflows/main.yml
-[version-badge]: https://img.shields.io/npm/v/@callstack/repack?style=flat-square
-[version]: https://www.npmjs.com/package/@callstack/repack
-[license-badge]: https://img.shields.io/npm/l/@callstack/repack?style=flat-square
-[license]: https://github.com/callstack/repack/blob/master/LICENSE
-[prs-welcome-badge]: https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square
+[callstack-readme-with-love]: https://callstack.com/?utm_source=github.com&utm_medium=referral&utm_campaign=repack&utm_term=readme-with-love
```

Comment:
this is unused now

Reply:
used here: https://github.com/callstack/repack/blob/776715a6d4ffcd60d51519cc3e0261d9f00be1a3/packages/repack/README.md?plain=1#L24